classes/Device.class.php

```diff
@@ -366,7 +366,7 @@ static function OSS_SNMP_Lookup($dev,$snmplookup,$oid=null,$walk=false){
 		$snmpHost=new OSS_SNMP\SNMP($dev->PrimaryIP,$dev->SNMPCommunity,$dev->SNMPVersion,$dev->v3SecurityLevel,$dev->v3AuthProtocol,$dev->v3AuthPassphrase,$dev->v3PrivProtocol,$dev->v3PrivPassphrase);
 		$snmpresult=false;
 		try {
-			$snmpresult=(is_null($oid))?$snmpHost->useSystem()->$snmplookup(true):($walk)?$snmpHost->realWalk($oid):$snmpHost->get($oid);
+			$snmpresult=((is_null($oid))?$snmpHost->useSystem()->$snmplookup(true):($walk))?$snmpHost->realWalk($oid):$snmpHost->get($oid);
```

Comment:
For the short amount of time I was trying to run php8 on my dev setup, these nested ternary conditionals were throwing fatal errors (PHP8 requires parens to avoid ambiguity).  I left this in the PR in case it is desired as a step towards PHP8 compat, otherwise I can revert it upon request.